### PR TITLE
Shared requests session across all test modules

### PR DESCRIPTION
**Pull request by rohitkadam19**
Used singleton approach to create only instance of requests session.

### CODE REVIEW

File: tests/api/test_api_register.py [hunk 1]
```diff
@@ -3,17 +3,16 @@
...
 
 class RegisterationAPITestCase(unittest.TestCase):
 
     @classmethod
     def setup_class(self):
         self.setting = Settings()
-        self.session = requests.session()
-        self.session.cert = self.setting.get_certs()
-        self.session.verify = False
+        ses_instance = Session()
```
Comment (girltravels) [inline tests/api/test_api_register.py]: please rename the var ses_instance to session_instance at all occurrences.

File: tests/api/test_group_api.py [hunk 2]
```diff
@@ -4,15 +4,15 @@
...
 
 
 class GroupsAPI(unittest.TestCase):
     def setup_class(self):
         self.setting = Settings()
-        self.session = requests.session()
-        self.session.cert = self.setting.get_certs()
-        self.session.verify = False
+        ses_instance = Session()
+        self.session = ses_instance.get_session()
```
Comment (anarang) [inline tests/api/test_group_api.py]: please rename the var ses_instance to session_instance at all occurrences.

**Comment by anarang** [comment]
@rohitkadam19 please squash the commits. You should have done 'git commit --amend' after making the changes as per the comment and then done a git push.

**Comment by rohitkadam19** [comment]
Closing this pull request and creating new one.

**Pull request closed without merging**